concepts/traits/about.md

```diff
@@ -144,23 +144,22 @@ fn main() {
         content: "Cairo is a new programming language for zero-knowledge proofs",
     };
 
-    println!("New article available! {}", news.summarize());
-}
-```
+    println!("New article available! {}", news.summarize()); // prints "New article available! (Read more...)"
 
-The default implementation returns `"(Read more...)"` ByteArray when this trait is implemented on any type. To use this default, you declare the impl block with an empty block.
 
-```rust
-impl NewsArticleSummary of Summary<NewsArticle> {}
+    let tweet = Tweet {
+        username: "Alice",
+        content: "Cairo is fun and engaging.",
+        reply: true,
+        retweet: true,
+    }
```

Comment:
The example does not compile due to the missing semicolon `;` here and the missing `Tweet` import. Add it and format the code.